### DISCUSSION TITLE
sub version release?

### DISCUSSION THREAD

**Discussion by remysheppard** [imfing/hextra#65] [General]
Can we package the most recent commit as 2.8.1 for a release? I've got the theme in as a mod, but it won't pull the image card update and I'm guessing that's why. Though, who knows, maybe there's another issue I'm having.

Still, I've tried uninstalling and reinstalling the mod, and my image cards aren't working.

**Comment by imfing** [comment]
@remysheppard I've just published a new release https://github.com/imfing/hextra/releases/tag/v0.2.9 
please update the theme and try again

**Reply by imfing** [reply]
you can also use `hugo mod get -u github.com/imfing/hextra@main` to use the latest main branch for your site

**Reply by remysheppard** [reply]
Thank you, I didn't know that! I'm honestly kind of new to all this, so thank you for your patience.

**Reply by imfing** [reply]
No problem, thanks for the contribution 😉

**Comment by remysheppard** [comment]
Works like a charm.
